### ISSUE TITLE
Sass sourcemaps not working with BrowserSync proxy

### ISSUE THREAD

**Issue by reintroducing** [floridoo/gulp-sourcemaps#90]
Hello,
I'm trying to get down to the bottom of the issue in the title and I'm not sure which task is at fault here, be it BrowserSync, gulp-sass, or gulp-sourcemaps.  I think it's gulp-sourcemaps but if not then please let me know.  I'll explain the issue as best as I can here in hopes of being led in the right direction:

We have a Django app running on test.local:8000. I'm using BrowserSync to proxy this and eventually to get proper BS my final URL ends up being test.local:3000 when I pull it up in the browser.  This now causes sourcemaps to not work correctly.  I can see the map file being created and it all looks correct but when I inspect my CSS I see it pointing to my CSS file not the Sass.  

I know that normally my setup is correct because when not using a proxy (in another project I'm just running localhost:3000) everything works as expected.    Interestingly enough, when using the proxy, even inline sourcemapping is not working, it's still showing me my CSS file in the inspector as the source.

Any ideas on why this would be failing?  I'd greatly appreciate any help you can offer.

**Comment by heikki** [comment]
If the sourcemap file is correct this can't be gulp-sourcemaps issue. Test the *same project* without proxy to make sure that it is the cause for the problem.

**Comment by reintroducing** [comment]
That's a good point, I'm not sure why I didn't think to run it without the proxy.  I'll change the topic of this ticket.

That being said, I've removed BS out of the equation all together and it's still not working.  I'm now just running a python server and my inspector still shows just the CSS file as the path back on any element versus the Sass file it originated in.  I tried both inline and separate file.

Here is my gulp task (again, this task works fine in another project where I'm developing locally not on a python server):

```
gulp.task('sass', function() {
    return gulp.src(config.sass + '/**/*.scss')
        .pipe(sourcemaps.init())
        .pipe(sass())
        .pipe(autoprefixer({
            browsers: ['Firefox > 25', 'last 4 versions', '> 3%', 'ie 8']
        }))
        .pipe(sourcemaps.write())
        .on('error', handleErrors)
        .pipe(gulp.dest(config.css));
});
```

Appreciate you taking time to help thus far.  Let me know if you see anything here that is not correct.
